test/e2e/common_test.go

```diff
@@ -130,26 +131,38 @@ var (
 )
 
 const (
-	// lockdir - do not use directly use LockTmpDir
+	// lockdir - do not use directly; use LockTmpDir
 	lockdir = "libpodlock"
 	// imageCacheDir - do not use directly use ImageCacheDir
 	imageCacheDir = "imagecachedir"
 )
 
 var _ = SynchronizedBeforeSuite(func() []byte {
-	globalTmpDir := GinkgoT().TempDir()
+	// One global scratch directory under which all test files will live.
+	// The usual case is that these tests are running in CI, on VMs
+	// with limited RAM, so we use /var/tmp.
+	baseTmpDir := "/var/tmp"
+	if os.Getenv("CI") == "" {
+		// Almost certainly a manual run, e.g., a developer with
+		// a hotrod workstation. Assume they know what they're doing.
+		baseTmpDir = ""
+	}
```

Comment:
I am fine to keep this for now

Reply:
CI VMs don't have enough RAM to run these tests in tmpfs. If we choose to experiment with super-duper RAM VMs, that's groovy, and it can easily be layered on top of this PR. In fact it can't even be started without this PR, because this PR is what makes `/tmp` be tmpfs.

All other feedback addressed, thank you.

Reply:
Drive-by suggestion: I don't think we need the entire `/var/tmp` on a ramdisk.  Might be worth while using some diskspace tracking tool to see what the `/var/tmp/subtest-*` consumes over time.  Maybe we don't require as big super-duper RAM VMs as we think?  Also, improvements to test-cleanup code could help out a lot here.

Reply:
> CI VMs don't have enough RAM to run these tests in tmpfs. If we choose to experiment with super-duper RAM VMs, that's groovy, and it can easily be layered on top of this PR. In fact it can't even be started without this PR, because this PR is what makes /tmp be tmpfs.

Which is why I said I am fine with this for now.

I don't suggest making /var/tmp tmpfs, we simple need to switch back to /tmp as root. This PR has the nessesary ground work to easily switch the basedir so we can experiment with it when it is merged